### ISSUE TITLE
1 second delay when opening a folder

### ISSUE THREAD

**Issue by starkgate** [files-community/Files#5452]
**Explain the bug**
Files has around a 1 second delay when opening a folder or a new tab. In comparison, Windows Explorer opens the same folders instantaneously. Is this something that is expected? Is performance something that can be improved in the near future?

**Logfile**
[debug.txt](https://github.com/files-community/Files/files/6840675/debug.txt)
[debug_fulltrust.txt](https://github.com/files-community/Files/files/6840676/debug_fulltrust.txt)

**System Information**
 - W10 x64 10.0.19043 Build 19043
 - 1.5.0.0

**Comment by gave92** [comment]
Yes! The beta version of Files is quite a bit quicker. If you wish to try it you can join Files insiders.
